src/CoilMaps.jl

```diff
@@ -80,6 +71,10 @@ function calcCoilMaps(data::AbstractVector{<:AbstractMatrix{Complex{T}}}, trj::A
     return cmaps
 end
 
-function calcCoilMaps(data::AbstractMatrix{Complex{T}}, trj::AbstractMatrix{T}, img_shape::NTuple{N,Int}; kernel_size=ntuple(_ -> 6, N), calib_size=ntuple(_ -> 24, N), eigThresh_1=0.01, eigThresh_2=0.9, nmaps=1, verbose=false) where {N,T}
-    calcCoilMaps([data], [trj], img_shape; U=I(1), kernel_size, calib_size, eigThresh_1, eigThresh_2, nmaps, verbose)
-end
+# wrappers for use with 4D arrays where the nr of ADC samples per readout is within a separate 2ⁿᵈ axis
+function calculateCoilMaps(data::AbstractArray{Tc,4}, trj::AbstractArray{T,4}, img_shape::NTuple{N,Int}; mask=trues(size(trj)[2:end]), kwargs...) where {T, Tc <: Complex, N}
+    data = reshape(data, :, size(data,3), size(data,4))
+    trj = reshape(trj, size(trj, 1), :, size(trj,4))
+    mask = reshape(mask, :, size(mask,3))
+    return calculateCoilMaps(data, trj, img_shape; kwargs..., mask)
```

Comment:
`U=CUDA.ones(T, size(trj)[end])` is a bug since this is CPU code. Is `U=I(size(trj)[end])` correct here?